sms.info.yml

```diff
@@ -8,3 +8,5 @@ configure: admin/config/smsframework
 dependencies:
   - telephone
   - dynamic_entity_reference
+test_dependencies:
```

Comment:
AFAIK, this particular setting is not taken from the patch, but from the repository.